pkg/iss/iss.go

```diff
@@ -828,7 +827,7 @@ func (iss *ISS) initOrderers() *events.EventList {
 		// Create segment.
 		seg := &orderers.Segment{
 			Leader:     leader,
-			Membership: iss.epoch.getNodeIDs(),
+			Membership: maputil.GetKeys(iss.epoch.nodeIDs),
```

Comment:
Actually, I was wrong with `maputil.GetKeys`. You need to use `maputil.GetSortedKeys`, since the order matters in the membership. Different nodes seeing a different memberhip order leads to some of the nastiest bugs.

Reply:
Resolved, but just FYI `maputil.GetKeys(membershipPb.Membership)` is being used still in the `Membership` function (`types.go`)